Description: Fix crash at launch time
 Patch is already available in master branch in upstream.
 Problem was only in builds with gcc 4.7.
Author: Pino Toscano <pino@debian.org>
Last-Update: 2012-05-12

--- a/eiskaltdcpp-qt/src/EiskaltApp.h
+++ b/eiskaltdcpp-qt/src/EiskaltApp.h
@@ -87,7 +87,7 @@ private:
 class EiskaltApp: public QtSingleCoreApplication {
 Q_OBJECT
 public:
-    EiskaltApp(int argc, char *argv[], const QString& uniqKey): QtSingleCoreApplication(argc, argv, uniqKey)
+    EiskaltApp(int &argc, char *argv[], const QString& uniqKey): QtSingleCoreApplication(argc, argv, uniqKey)
     {
         installEventFilter(&ef);
     }
--- a/eiskaltdcpp-qt/src/EiskaltApp_haiku.h
+++ b/eiskaltdcpp-qt/src/EiskaltApp_haiku.h
@@ -87,7 +87,7 @@ private:
 class EiskaltApp: public QApplication {
 Q_OBJECT
 public:
-    EiskaltApp(int argc, char *argv[], const QString& uniqKey): QApplication(argc, argv)
+    EiskaltApp(int &argc, char *argv[], const QString& uniqKey): QApplication(argc, argv)
     {
         installEventFilter(&ef);
     }
